.github/workflows/main.yaml

```diff
@@ -7,6 +7,9 @@ on:
     tags:
   pull_request:
 
+env:
+  CONTAINER_RUN_OPTIONS: " "
```

Comment:
```suggestion
  CONTAINER_RUN_OPTIONS: ""
```

Reply:
if fixed like this, this means env is not set,  and will be set to default -it